src/pharmpy/modeling/common.py

```diff
@@ -734,3 +734,17 @@ def get_model_attributes(model: Model, supress_warnings=False) -> str:
             if (e is not None and e != 0)
         ]
     )
+
+
+def filter_dataset(model, dv=1):
```

Comment:
This can be made more general. There is a cli function that filters the dataset given a general expression. We could do the same here.